js/jquery.mapael.js

```diff
@@ -1736,6 +1743,29 @@
                 }, interval
             );
         },
+        
+        /*
+          * Check for Raphael bug regarding drawing while beeing hidden (under display:none)
+          * See https://github.com/neveldo/jQuery-Mapael/issues/135
+          * @return true/false
+          *
+          * Wants to override this behavior? Use prototype overriding:
+          *     $.mapael.prototype.isPaperHidden = function() {return false;};
+          */
+        isPaperHidden: function(){
```

Comment:
I wonder if a check like self.$container.is(":visible"); wouldn't be enough here ?